extensions/ql-vscode/src/extension.ts

```diff
@@ -70,12 +70,12 @@ import { asError, assertNever, getErrorMessage } from './pure/helpers-pure';
 import { spawnIdeServer } from './ide-server';
 import { InterfaceManager } from './interface';
 import { WebviewReveal } from './interface-utils';
-import { ideServerLogger, logger, queryServerLogger } from './logging';
+import { ideServerLogger, logger, ProgressReporter, queryServerLogger } from './logging';
 import { QueryHistoryManager } from './query-history';
 import { CompletedLocalQueryInfo, LocalQueryInfo } from './query-results';
-import * as qsClient from './queryserver-client';
+import * as qsClient from './legacy-query-server/queryserver-client';
+import * as qsClient2 from './query-server/queryserver-client';
```

Comment:
Maybe this is more understandable?

```suggestion
import * as legacyQsClient from './legacy-query-server/queryserver-client';
import * as newQsClient from './query-server/queryserver-client';
```